### DISCUSSION TITLE
How to dynamically build sidebars using frontmatter

### DISCUSSION THREAD

**Discussion by audreyshub** [vuejs/vitepress#1416] [Q&A] [answered]
👋 I want different sidebars based on different navs. For example, I have 3 different products on my site, and so I want 3 different sidebars — each being unique for that product. 

I can store the name of the sidebar in the page frontmatter but I’m not sure how I can pass that value into the sidebar function in  `config.js` to build the sidebar dynamically? Could this be done and, if so, how? 

Thank you 🙏

**Comment by emersonbottero** [comment, marked as answer]
https://github.com/Stuyk/vitepress-sidebar-builder
